### ISSUE TITLE
Missing label on form element

### ISSUE THREAD

**Issue by imperator99** [govau/dta-gov-au#130]
The Chosen `<input>` element is missing a `label` attribute.

**Comment by imperator99** [comment]
Resolved by https://github.com/govau/dta-gov-au/pull/151.
